tools/deb/BUILD

```diff
@@ -2,7 +2,7 @@ load("@bazel_tools//tools/build_defs/pkg:pkg.bzl", "pkg_tar", "pkg_deb")
 
 pkg_tar(
     name = "agent-bin",
-    files = [
+    srcs = [
```

Comment:
revert this ?
```
W0915 01:32:57.236] ERROR: /home/bootstrap/go/src/github.com/istio/pilot/tools/deb/BUILD:3:1: //tools/deb:agent-bin: no such attribute 'srcs' in 'pkg_tar' rule.
```

Reply:
Oddly enough bazel produces a warning in my environment that `files` doesn't exist anymore (deprecated?) and that users should switch to `srcs`. This is consistent with bazel document here  https://docs.bazel.build/versions/master/be/pkg.html#pkg_tar.

Reply:
probably you are using a different bazel version than the prow one - for now prow wins (might be a great idea to make a new prow image, it's in test-infra / or ask the oncall)